test/SFForceJSTestSuite.js

```diff
@@ -226,6 +226,29 @@ ForceJSTestSuite.prototype.testRestEndpoint = function()  {
      });
 };
 
+/**
+ * TEST restEndpointErrorIOS
+ */
+ForceJSTestSuite.prototype.testRestEndpointErrorIOS = function()  {
```

Comment:
What happens on Android?

Reply:
It'll return the `Exception` that's thrown from the native network layer, if I'm not mistaken.
```
AsyncRequestCallback:

		void onError(Exception exception);
```

Reply:
Could we have a js test that runs on both platform?

Reply:
I did the iOS one to make sure the three components of the dictionary are there, but I could make it more generic and just assert that an error exists so that it would work for both platforms

Reply:
That would be nice (and you would have to add the new test in https://github.com/forcedotcom/SalesforceMobileSDK-Android/blob/dev/libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ForceJSTest.java).